win32.mak

```diff
@@ -30,11 +30,6 @@ d.chm : chm\d.chm
 chm-nav-release.json : $(DDOC) std.ddoc spec\spec.ddoc modlist-release.ddoc changelog\changelog.ddoc chm-nav.dd
 	$(DMD) -o- -c -Df$@ $**
 
-d.tag : chmgen.exe $(TARGETS)
-	chmgen --only-tags
-
-d-tags.json : d.tag
-
```

Comment:
Why? Is this obsoleted by posix.mak? Are we sure this is not used anywhere, incl. Martin's release-building scripts?

Reply:
Yes, `chmgen` already depends on a Linux host to build the HTML files, for which the `d.tags.json` is built too (in fact https://github.com/dlang/installer/pull/298 shows that this already has been done).
Apart from that building the `d-tags.json` comes with its problems (e.g. root of the HTML files etc.), so I thought it's best to keep only what's really run.

>  incl. Martin's release-building scripts?

Yes, as mentioned he copied the JSON file over from Linux.
See also:

https://github.com/dlang/installer/blob/master/create_dmd_release/create_dmd_release.d#L514
https://github.com/dlang/installer/pull/298